Detections/SymantecProxySG/ExcessiveDeniedProxyTraffic.yaml

```diff
@@ -0,0 +1,26 @@
+id: 7a58b253-0ef2-4248-b4e5-c350f15a8346
+name: Excessive Denied Proxy Traffic
+description: |
+  'This alert creates an incident when a client generates an excessive amounts of denied proxy traffic.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Syslog
+    dataTypes: 
+      - SymantecProxySG
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - DefenseEvasion
+relevantTechniques:
+  - T1090
+query: |
+
+  let timeframe = ago(1h);
+  SymantecProxySG 
+  | where TimeGenerated >= timeframe
+  | where sc_filter_result == "DENIED"
+  | summarize count() by c_ip, cs_host
```

Comment:
bring thru time and to not cause issues with your count using a bin(), lets just get the min() and max() in the summarize then we can use these in the timestamp custom entity required for detections. In this case -
| summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), count() by c_ip, cs_host

Reply:
Corrections made and pushed